<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_main</class>
 <widget class="QMainWindow" name="w_main">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lb_versions">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#000000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>561</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bt_launch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: none</string>
         </property>
         <property name="text">
          <string>Launch Civilization IV</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_components">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>See latest changes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bt_option">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: none</string>
         </property>
         <property name="text">
          <string>Options</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="flag">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuFix_installation">
     <property name="title">
      <string>Fix installation</string>
     </property>
     <addaction name="actionClear_cache"/>
     <addaction name="actionReset"/>
     <addaction name="actionApply_Asian_language_patch_again"/>
    </widget>
    <addaction name="actionOpen_mod_folder"/>
    <addaction name="menuFix_installation"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommunity">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Community</string>
    </property>
    <widget class="QMenu" name="menuAddons">
     <property name="title">
      <string>Modules</string>
     </property>
     <widget class="QMenu" name="menuAddons_list">
      <property name="title">
       <string>Get modules</string>
      </property>
     </widget>
     <addaction name="menuAddons_list"/>
     <addaction name="separator"/>
     <addaction name="actionDetected"/>
    </widget>
    <addaction name="actionWebsite"/>
    <addaction name="actionForum"/>
    <addaction name="actionAbout_AND_Resurrection_team_forum"/>
    <addaction name="actionDevelopment_platform"/>
    <addaction name="menuAddons"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Help us to improve the mod !</string>
    </property>
    <widget class="QMenu" name="menuTranslation">
     <property name="title">
      <string>Translation</string>
     </property>
     <addaction name="actionHelp_translate_the_mod"/>
     <addaction name="actionTranslate_the_civilopedia"/>
     <addaction name="actionTranslate_the_website"/>
     <addaction name="actionTranslate_the_module_Mega_Civ_pack"/>
     <addaction name="actionTranslate_the_mod_help"/>
    </widget>
    <addaction name="menuTranslation"/>
    <addaction name="actionBugreport"/>
    <addaction name="actionGive_us_feedback_forum"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="language_ar"/>
    <addaction name="language_zh"/>
    <addaction name="language_cs"/>
    <addaction name="language_da"/>
    <addaction name="language_en"/>
    <addaction name="language_fi"/>
    <addaction name="language_fr"/>
    <addaction name="language_de"/>
    <addaction name="language_hu"/>
    <addaction name="language_it"/>
    <addaction name="language_ja"/>
    <addaction name="language_ko"/>
    <addaction name="language_pl"/>
    <addaction name="language_ru"/>
    <addaction name="language_es"/>
    <addaction name="language_tr"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuCommunity"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum</string>
   </property>
  </action>
  <action name="actionBugreport">
   <property name="text">
    <string>Report a bug (forum)</string>
   </property>
  </action>
  <action name="actionAbout_AND_Resurrection_team_forum">
   <property name="text">
    <string>History and credits</string>
   </property>
   <property name="toolTip">
    <string>About AND Resurrection team (forum)</string>
   </property>
  </action>
  <action name="actionOpen_mod_folder">
   <property name="text">
    <string>Open mod folder</string>
   </property>
  </action>
  <action name="actionHelp_translate_the_mod">
   <property name="text">
    <string>Translate the mod</string>
   </property>
  </action>
  <action name="actionTranslate_the_mod_help">
   <property name="text">
    <string>Help for translation (forum)</string>
   </property>
  </action>
  <action name="language_en">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="language_fr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="language_hu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hungarian</string>
   </property>
  </action>
  <action name="language_fi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Finnish</string>
   </property>
  </action>
  <action name="language_pl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polish</string>
   </property>
  </action>
  <action name="language_it">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Italian</string>
   </property>
  </action>
  <action name="language_es">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="language_de">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="language_ru">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionClear_cache">
   <property name="text">
    <string>Clear cache</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset mod to default state</string>
   </property>
  </action>
  <action name="actionTranslate_the_civilopedia">
   <property name="text">
    <string>Translate the civilopedia</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="language_cs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Czech</string>
   </property>
  </action>
  <action name="language_da">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Danish</string>
   </property>
  </action>
  <action name="language_ar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arabic</string>
   </property>
  </action>
  <action name="language_tr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Turkish</string>
   </property>
  </action>
  <action name="actionDetected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Activation status:</string>
   </property>
  </action>
  <action name="actionGive_us_feedback_forum">
   <property name="text">
    <string>Give us feedback ! (forum)</string>
   </property>
  </action>
  <action name="actionTranslate_the_website">
   <property name="text">
    <string>Translate the website</string>
   </property>
  </action>
  <action name="actionTranslate_the_module_Mega_Civ_pack">
   <property name="text">
    <string>Translate the module &quot;Mega Civ. pack&quot;</string>
   </property>
  </action>
  <action name="language_ko">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="language_zh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="language_ja">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
  <action name="actionDevelopment_platform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Development tracker</string>
   </property>
  </action>
  <action name="actionApply_Asian_language_patch_again">
   <property name="text">
    <string>Apply Asian language patch again</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
